Platforms/QemuQ35Pkg/CpuInfoDxe/CpuInfoDxe.c

```diff
@@ -0,0 +1,129 @@
+/** @file CpuInfoDxe.c
+
+    Copyright (c) Microsoft Corporation. All rights reserved.
+
+    Driver that prints CPU branding information
+**/

```

Comment:
I know older style placed the description below the copyright, but can you please place it above in new files?

`SPDX-License-Identifier: BSD-2-Clause-Patent` is also needed in all files.

```suggestion
/** @file CpuInfoDxe.c
    DXE driver that prints CPU branding information
    
    Copyright (c) Microsoft Corporation. All rights reserved.

    SPDX-License-Identifier: BSD-2-Clause-Patent
**/

```